other-tabs/collaboration/distribute-tasks-with-inboxes.md

```diff
@@ -0,0 +1,97 @@
+
+You can then select between two dispatch rules:
+
+- "Field based":
+  - You can select up to 3 fields.
+  - For each field, you can sort ascending or descending.
+  - The records will be sorted by the first sorting field, in case of equality, the second sorting field is applied and so on to the third sorting field.
+  - For example:
+    - You can select the first sorting field to be the create date of the document, sorted descending
+    - You can select the second sorting field to be the importance (not sure for this one) of the document, sorted ascending
+    - For two `Documents` with the same creation date, the importance will take over for the first that will be assigned
+- "Random":
+  - The `Documents` will randomly be assigned
+
+## Processing tickets
+
+Once an inbox has been created, access the inbox from the "Collaboration" tab.
+
+### Todo section
+
+The count near the "Todo" section represents the number of records currently in the collection segment selected for the inbox (e.g. `Waiting for Validation`) that have not yet been assigned to an operator.
+
+In the "Todo" section of the inbox, an operator can click on the "Start processing button" to have a task automatically assigned, and being redirected on the record.
+
+The user then does what is necessary to achieve the current workflow, for example trigger a `Validate Document` action. Then the user can click the "Next ticket" button on the top of Forest Admin UI, to be assigned to a new record.
+
+### Doing section
+
+On the "Doing" section, the user can view the tasks that are still in the segment and that are currently assigned to he or she.
+
+### Backlog section
+
+This section is accessible for users with a Editor or Administrator role.
+
+It provides an overview of the current inbox.
+
+You can see all the records in the segment their current task state, and which user is assigned to it.
+
+For the "Doing" tasks, you can see the date when the task was started.
+
+For the "Completed" tasks, you can also see the time it took for the task to get out of the segment. If the records gets out of the segment using an action requiring an approval, it includes the approval time.
+
+### Manual task assignment
+
+Assigning a task to an operator, makes this record be the next one to be assigned for this user, bypassing the sorting rules of the inbox.
+
+It is possible to manually assign a task to a specific user from the [backlog](#backlog-section).
+
+It is also possible to manually assign a task to an operator from the record view, through the native Forest Admin action "Assign to...".
+
+From this action, you can assign a task to users that are not in you current team and also select the inbox. The team should have an inbox based on a segment matching the record state.
+
+## Inbox settings
+
+Inbox settings are accessible from the Edit layout mode.
+
+You can rename and delete inboxes from the settings.
```

Comment:
and configure options like maximum doing records per operator and duration before auto unassign

Reply:
Ok I did not see the others parts. Can we have a big section with title inside ? 
Or rename this section name ?

Reply:
> and configure options like maximum doing records per operator and duration before auto unassign
I didn't mention these one because here I only speak about basic features not detailed later, same for the folders

Reply:
Yes but you should specify it (for example: "and more, see below...")